packages/devtools_app/lib/src/screens/memory/panes/leaks/LEAK_TRACKING.md

```diff
@@ -116,6 +116,34 @@ like [the example app does](https://github.com/polina-c/spikes/blob/master/leaki
 
 You can pass information, that will help you troubleshoot the leak, to the optional parameter `details`.
 
+TODO(polina-c): explain how to choose which classes to instrument.
+
+### Detect leaks in your Dart app
+
+For Dart appliocations: 
+
+1. Reference [memory_leak_tracker](https://github.com/polina-c/spikes/blob/master/memory_leak_tracker/README.md):
```

Comment:
are users supposed to add this to their pubspec.yaml file? if so, let's make that clear here